### PR TITLE
IGNITE-13285

### PR DESCRIPTION
**Pull request by Nikita-tech-writer**
Thank you for submitting the pull request to the Apache Ignite.

In order to streamline the review of the contribution 
we ask you to ensure the following steps have been taken:

### The Contribution Checklist
- [ ] There is a single JIRA ticket related to the pull request. 
- [ ] The web-link to the pull request is attached to the JIRA ticket.
- [ ] The JIRA ticket has the _Patch Available_ state.
- [ ] The pull request body describes changes that have been made. 
The description explains _WHAT_ and _WHY_ was made instead of _HOW_.
- [ ] The pull request title is treated as the final commit message. 
The following pattern must be used: `IGNITE-XXXX Change summary` where `XXXX` - number of JIRA issue.
- [ ] A reviewer has been mentioned through the JIRA comments 
(see [the Maintainers list](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute#HowtoContribute-ReviewProcessandMaintainers)) 
- [ ] The pull request has been checked by the Teamcity Bot and 
the `green visa` attached to the JIRA ticket (see [TC.Bot: Check PR](https://mtcga.gridgain.com/prs.html))

### Notes
- [How to Contribute](https://cwiki.apache.org/confluence/display/IGNITE/How+to+Contribute)
- [Coding abbreviation rules](https://cwiki.apache.org/confluence/display/IGNITE/Abbreviation+Rules)
- [Coding Guidelines](https://cwiki.apache.org/confluence/display/IGNITE/Coding+Guidelines)
- [Apache Ignite Teamcity Bot](https://cwiki.apache.org/confluence/display/IGNITE/Apache+Ignite+Teamcity+Bot)

If you need any help, please email dev@ignite.apache.org or ask anу advice on http://asf.slack.com _#ignite_ channel.

### CODE REVIEW

File: docs/_docs/tools/control-script.adoc [hunk 1]
```diff
@@ -646,4 +646,50 @@ control.bat --change-tag _<new-tag>_
...
 
+== Indexes Management
+
+The commands below allow to get a specific information on indexes and to trigger the indexes rebuild process.
+
+To get the list of all indexes that match specified filters, use the command:
+
+[source, shell]
+----
+--cache indexes_list [--node-id nodeId] [--group-name grpRegExp] [--cache-name cacheRegExp] [--index-name idxNameRegExp]
```
Comment (dmagda) [inline docs/_docs/tools/control-script.adoc:657]: The command is incomplete. There needs to be a version for Unix (control.sh) and Windows (control.bat). See how other commands are explained above.

File: docs/_docs/tools/control-script.adoc [hunk 2]
```diff
@@ -646,4 +646,50 @@ control.bat --change-tag _<new-tag>_
...
+| `--group-name regExp`| Regular expression enabling filtering by cache group name.
+| `--cache-name regExp`| Regular expression enabling filtering by cache name.
+| `--index-name regExp`| Regular expression enabling filtering by index name.
+|===
+
+To get the list of all caches that have index rebuild in progress, use the command below:
+
+[source, shell]
+----
+--cache indexes_rebuild_status [--node-id nodeId]
```
Comment (dmagda) [inline docs/_docs/tools/control-script.adoc:675]: Same here and down below. The command needs to start with a script name (control.sh or control.bat) and only then you pass in the parameters. Btw, try to run it, to make sure it works. That's required.

File: docs/_docs/tools/control-script.adoc (hunk 1 above)
Comment (Nikita-tech-writer) [reply docs/_docs/tools/control-script.adoc]: Fixed.

File: docs/_docs/tools/control-script.adoc (hunk 2 above)
Comment (Nikita-tech-writer) [reply docs/_docs/tools/control-script.adoc]: Fixed.

**Comment by Nikita-tech-writer** [comment]
@dmagda I've fixed both issues. Please take a look once again.

File: docs/_docs/tools/control-script.adoc [hunk 3]
```diff
@@ -646,4 +646,208 @@ control.bat --change-tag _<new-tag>_
 ----
 --
 
+== Metric Command
+
+The metrics command prints out the value of a metric or metric registry provided in the parameters list. Use the `--node-id` parameter, If you need to get a metric from a specific node. Ignite selects a random node, if the `--node-id` is not set.
```
Comment (dmagda) [inline docs/_docs/tools/control-script.adoc:651]: "If you need to" -> "if you need to". The first letter needs to be in the lower case.

**Review by dmagda — COMMENTED** [review]
@Nikita-tech-writer, a minor missprint. Plus, GitHub says there is some merge conflict. Update the recent changes from the master before updating the pull-request.

File: docs/_docs/tools/control-script.adoc (hunk 3 above)
Comment (Nikita-tech-writer) [reply docs/_docs/tools/control-script.adoc]: Fixed.

**Comment by Nikita-tech-writer** [comment]
@dmagda Please verify my latest changes.

**Pull request merged**